tests/MessageTest.php

```diff
@@ -594,4 +594,38 @@ public function testMultipartMessageWithoutCharset()
         $this->assertSame('MyPlain', \rtrim($message->getBodyText()));
         $this->assertSame('MyHtml', \rtrim($message->getBodyHtml()));
     }
+
+    public function testGetInReplyTo()
+    {
+        $fixture = $this->getFixture('references');
+        $this->mailbox->addMessage($fixture);
+
+        $message = $this->mailbox->getMessage(1);
+
+        $this->assertCount(1, $message->getInReplyTo());
+
+        $fixture = $this->getFixture('plain_only');
+        $this->mailbox->addMessage($fixture);
+
+        $message = $this->mailbox->getMessage(2);
+
+        $this->assertCount(0, $message->getInReplyTo());
+    }
+
+    public function testGetReferences()
+    {
+        $fixture = $this->getFixture('references');
+        $this->mailbox->addMessage($fixture);
+
+        $message = $this->mailbox->getMessage(1);
+
+        $this->assertCount(2, $message->getReferences());
```

Comment:
Please add also specific assertion to the content of the referencies, e.g. `$this->assertContains('<231d9ac57aec7d8c1a0eacfeab8af6f3@example.com>, $message->getReferences());'`